src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -287,11 +288,14 @@ public T WithLegacyCacheCompatibility(bool enableLegacyCacheCompatibility = true
         /// You can set it to <c>true</c> for advanced debugging requiring PII
         /// If both WithLogging apis are set, this one will override the other
         /// </param>
-        /// <returns></returns>
+        /// <returns>The builder to chain the .With methods</returns>
+        /// <item><description>This is an experimental API. The method signature may change in the future without involving a major version upgrade.</description></item>
```

Comment:
nit:  remarks instead?
```suggestion
        /// <remarks>This is an experimental API. The method signature may change in the future without involving a major version upgrade.</remarks>
```